resources.go

```diff
@@ -1424,8 +1430,14 @@ func Provider() tfbridge.ProviderInfo {
 			"azurerm_private_link_endpoint_connection": {
 				Tok: azureDataSource(azurePrivateLink, "getPrivateLinkEndpointConnection"),
 			},
-			"azurerm_virtual_hub":     {Tok: azureDataSource(azureNetwork, "getVirtualHub")},
-			"azurerm_signalr_service": {Tok: azureDataSource(azureSignalr, "getService")},
+			"azurerm_virtual_hub":                 {Tok: azureDataSource(azureNetwork, "getVirtualHub")},
+			"azurerm_signalr_service":             {Tok: azureDataSource(azureSignalr, "getService")},
+			"azurerm_storage_container":           {Tok: azureDataSource(azureStorage, "getStorageContainer")},
+			"azurerm_iothub_shared_access_policy": {Tok: azureDataSource(azureIot, "getSharedAccessPolicy")},
+			"azurerm_iothub_dps":                  {Tok: azureDataSource(azureIot, "getDps")},
+			"azurerm_eventgrid_topic":             {Tok: azureDataSource(azureEventGrid, "getTopic")},
+			"azurerm_disk_encryption_set":         {Tok: azureDataSource(azureEventGrid, "getDiskEncryptionSet")},
+			"azurerm_dedicated_host_group":        {Tok: azureDataSource(azureEventGrid, "getDedicatedHostGroup")},
```

Comment:
These last two should be `compute` if I read it right

Reply:
great catch!